store/ssmstore.go

```diff
@@ -99,6 +107,10 @@ func (s *SSMStore) Write(ctx context.Context, id SecretId, value string) error {
 		version = current.Meta.Version + 1
 	}
 
+	if len(tags) > 0 && version != 1 {
+		return errors.New("tags on write only supported for new secrets")
+	}
```

Comment:
❓ Curious if there is a particular reason why tagging is only support for new secrets? Sorry if I am missing it somewhere

Reply:
You didn't miss anything! (I thought for a second that I had documented why, but it's just in my notes.) I was thinking of what chamber should do when the secret already exists and it already has tags.

* If you include `--tags`, what should happen to the other tags that you don't list? Should they be deleted or left alone? Admittedly, we could add the `--delete-other-tags` option from `chamber tag write` to let you pick.
* If you do not include `--tags`, what should happen to the existing tags? Should they be all dropped out or left alone? We could maybe add another option, `--delete-existing-tags` or something, to let you pick.

It's because of this complexity that, at least for this pass, I opted to restrict tags to just new secrets.

Reply:
thank you for the context! I did not know that it would override the existing tag prior.